lib/rmt.toit

```diff
@@ -296,9 +301,12 @@ The $rx channel must be configured for receiving (see $Channel.config_rx).
 The $tx channel must be configured for transferring (see $Channel.config_tx).
 */
 transfer_and_receive --rx/Channel --tx/Channel signals/Signals max_returned_bytes/int -> Signals:
-  if max_returned_bytes > rx.rx_buffer_size: throw "maximum returned buffer size greater than allocated RX buffer size"
+  if not rx.rx_buffer_size or not rx.rx_clk_div: throw "rx channel not configured"
 
-  result := rmt_transfer_and_read_ tx.num rx.num signals.bytes_ max_returned_bytes
+  if not rx.rx_buffer_size or max_returned_bytes > rx.rx_buffer_size: throw "maximum returned buffer size greater than allocated RX buffer size"
```

Comment:
We are rechecking rx.rx_buffer_size for nullness, which may not be what you want.

Reply:
Removed the second check.